### PR TITLE
[Release] Linux_SQL

### PR DESCRIPTION
**Pull request by Hunain-Khalid**
Releasing code from dev to release branches

### CODE REVIEW

File: linux_sql/.idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```
Comment (trevor-jrvs) [inline linux_sql/.idea/.gitignore:1]: git rm -r linux_sql/.idea

File: linux_sql/scripts/host_info.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+l2_cache=$(echo "$lscpu_out"  | egrep "L2 cache:" | awk '{print substr($3,1,length($3)-1)}' | xargs)
+total_mem=$(echo "$mem_info_out"  | egrep "MemTotal:" | awk '{print $2}' | xargs)
+timestamp=$(echo "$vmstat_info" | awk 'NR == 3 {print $18 " "$19}' | xargs)
+
+# Insert data into table, host_info using data below
+insert_data="INSERT INTO host_info (hostname, cpu_number, cpu_architecture, cpu_model, cpu_mhz, L2_cache, total_mem, timestamp) VALUES ('$hostname', $cpu_number, '$cpu_architecture', '$cpu_model', $cpu_mhz, $l2_cache, $total_mem, '$timestamp')"
+export PGPASSWORD=$psql_password
+#Insert data into a database, using psql instance
+psql -h $psql_host -p $psql_port -d $db_name -U $psql_user -c "$insert_data"
+exit 0
```
Comment (trevor-jrvs) [inline linux_sql/scripts/host_info.sh:38]: exit $?

File: linux_sql/scripts/host_usage.sh [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+cpu_idle=$(echo "$vmstat_info" | awk 'NR == 3 {print $15}'| tail -n1 |  xargs)
+cpu_kernel=$(echo "$vmstat_info" | awk 'NR == 3 {print $14}'| xargs)
+disk_io=$(echo "$vmstat_d_io" | awk 'NR == 3 {print $10}'| xargs)
+disk_available=$(echo "$df_stat" | awk 'NR == 2 {print substr($4,1,length($4)-1)}' | xargs)
+
+# Needed subquery to find id of host
+host_id="(SELECT id FROM host_info WHERE hostname='$hostname')";
+
+# Insert data into table, host_usage using data below
+insert_data="INSERT INTO host_usage (timestamp, host_id, memory_free, cpu_idle, cpu_kernel, disk_io, disk_available) VALUES ('$timestamp', $host_id, $memory_free, '$cpu_idle', '$cpu_kernel', $disk_io ,$disk_available)"
```
Comment (trevor-jrvs) [inline linux_sql/scripts/host_usage.sh:42]: consider breaking into multiple lines

File: linux_sql/sql/queries.sql [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    LANGUAGE PLPGSQL;
+
+-- query 1
+SELECT cpu_number, id, total_mem
+from host_info
+ORDER BY cpu_number, (total_mem) DESC;
+
+-- query 2
+select hu.host_id, hi.hostname, round5(hu.timestamp) as interval_5min,
+(
```
Comment (trevor-jrvs) [inline linux_sql/sql/queries.sql:17]: condense this

File: linux_sql/sql/queries.sql [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+$$
+BEGIN
+    RETURN date_trunc('hour', ts) + date_part('minute', ts):: int / 5 * interval '5 min';
+END;
+$$
+    LANGUAGE PLPGSQL;
+
+-- query 1
+SELECT cpu_number, id, total_mem
+from host_info
```
Comment (trevor-jrvs) [inline linux_sql/sql/queries.sql:12]: be consistent with capitalization

**Review by trevor-jrvs — APPROVED** [review]
lgtm

**Comment by Hunain-Khalid** [comment]
finished checking release on git, via develop branch, now pushing changes to release

**Pull request merged**